learn/teams/import-vcs-project.md

```diff
@@ -1,95 +1,83 @@
 ---
-title: Import a WaveMaker Project using VCS URL
+title: Import Project from Code Repository
 id: ""
 sidebar_label: "Import VCS Project"
 ---
 ---
 
-Create a WaveMaker project using the existing repository URL from VCS (Version Control Systems). For this, you will need VCS clone URL to create a project in WaveMaker Studio. 
+Create a WaveMaker project using the existing repository URL from VCS (Version Control Systems). For this, you will need VCS clone URL and its Credentials to create a project in WaveMaker Studio. Git History of the project also will be cloned along with project.
 
-**Supported VCS providers**: GitHub and GitLab
+Import from Code repository option is avilable in **Manage Projects** page In **Teamportal**.
+
+![import project button](/learn/assets/import-from-code-repository-button.png)
+
+**Supported VCS providers**: GitHub, GitLab, Bitbucket, Azure
 
 ## Importing VCS Project to WaveMaker Studio
 
 Typically, you may use this feature in the following scenarios:
 
-- To transfer projects from one VCS provider to another. 
+- Migrating projects from One VCS to Another VCS
+ 
 - To change the group/organization name of an existing repository.
```

Comment:
To change the group/organization name of an existing configured code repository.